### ISSUE TITLE
Strings for Onyx Note X2

### ISSUE THREAD

**Issue by amenekowo** [Hagb/decryptBooxUpdateUpx#52]
```
"MODEL" = "NoteX2",
"STRING_SETTINGS" = "CTvhm+r7t0WkCO0GSd3wsPz9XM9I8NPkCE01nWR8jbii",
"STRING_UPGRADE" = "eE6c45CLsjvPltIFTvnZjS7n9TX57Pergj9uRyAOFK4w"
# fingerprint: ONYX/BOOX/BOOX:11/RKQ1.210614.002/200:userdebug/release-keys
```

Latest UPX update URL: [update.upx](http://firmware-volc.boox.com/234c4943fb1fc3f3623897748ea937a8/update.upx)

By the way, where can I get the STRING_LOCAL used in DeBooxUpx.py ?
Thanks

**Comment by Hagb** [comment]
Sorry for my late reply.

> By the way, where can I get the STRING_LOCAL used in DeBooxUpx.py ?

AFAIK, recent models, whose strings are got by [GetBooxUpxKeysApp](https://github.com/Hagb/GetBooxUpxKeysApp/releases), don't have the `LOCAL` string. It was used by the updater app of older models to figure out the path to save decrypted plain update.zip.

**Comment by Hagb** [comment]
Thanks for your strings! They have been verified and merged in 1f1a01f.
